common/conf.py

```diff
@@ -150,22 +151,15 @@ def get_previous_version(majornumber, minornumber=0):
     'ome-users' : (lists_root + '/pipermail/ome-users/%s' ,''),
     'ome-devel' : (lists_root + '/pipermail/ome-devel/%s' ,''),
     'forum' : (oo_root + '/community/%s', ''),
-    # Plone links. Separating them out so that we can add prefixes and
-    # suffixes during testing.
-    'community_plone' : (oo_site_root + '/community/%s', ''),
-    'legacy_plone' : (oo_site_root + '/support/previous/%s', ''),
-    'about_plone' : (oo_site_root + '/about/%s', ''),
-    'team_plone' : (oo_site_root + '/team/%s', ''),
-    'schema_plone' : (oo_root + '/Schemas/%s', ''),
-    'omero_plone' : (oo_site_root + '/products/omero/%s', ''),
-    'secvuln' : (oo_root + '/info/vulnerabilities/%s', ''),
-    'bf_plone' : (oo_site_root + '/products/bio-formats/%s', ''),
-    'partner_plone' : (oo_site_root + '/products/partner/%s', ''),
-    'cpp_plone' : (oo_site_root + '/products/ome-files-cpp/%s', ''),
+    # Website links
+    'community' : (oo_root + '/support/%s', ''),
+    'omero' : (oo_root + '/omero/%s', ''),
+    'bf' : (oo_root + '/bio-formats/%s', ''),
     # One branch only doc links. Branched docs links in conf.py files for
     # individual doc sets
-    'model_doc' : (oo_site_root + '/support/ome-model/%s', ''),
-    'devs_doc' : (oo_site_root + '/support/contributing/%s', ''),
+    'model_doc' : (docs_root + '/ome-model/5.5.7/%s', ''),
```

Comment:
latest is in place for model docs yet so I've added a versioned link to fix the build